### ISSUE TITLE
32 character limit for side bar CRD size, large objects (long CRD names ) get cut off

### ISSUE THREAD

**Issue by jayunit100** [vmware-tanzu/octant#537]
# Preamble

Thank you so much for this wonderful project :) it saved my life today when navigating a sea of CRDs.. Here's the first issue i found ... im sure more to come.  Happy to contribute a fix but time is limited for me as im commited to other things for my day job. 

# what was expected 

Assuming arbitrarily long CRDs would be viewable/copyable from the octant sidebar.

# what was observed 

CRDs w longer then 32 characters are not viewable in octant's sidebar, making exploring a complex cluster w/ lots of CRDs a little tricky

# Deets 
In the UI, If you have long CRDs, like .
 
```
bgpconfigurations.crd.projectcalico.org                   2020-01-13T22:26:38Z
bgppeers.crd.projectcalico.org                            2020-01-13T22:26:37Z
blockaffinities.crd.projectcalico.org                     2020-01-13T22:26:37Z
clusterinformations.crd.projectcalico.org                 2020-01-13T22:26:38Z
clusters.cluster.x-k8s.io                                 2020-01-13T22:26:42Z
felixconfigurations.crd.projectcalico.org                 2020-01-13T22:26:37Z
globalnetworkpolicies.crd.projectcalico.org               2020-01-13T22:26:39Z
globalnetworksets.crd.projectcalico.org                   2020-01-13T22:26:39Z
hostendpoints.crd.projectcalico.org                       2020-01-13T22:26:38Z
ipamblocks.crd.projectcalico.org                          2020-01-13T22:26:37Z
ipamconfigs.crd.projectcalico.org                         2020-01-13T22:26:37Z
ipamhandles.crd.projectcalico.org                         2020-01-13T22:26:37Z
kubeadmconfigs.bootstrap.cluster.x-k8s.io                 2020-01-13T22:26:43Z
kubeadmconfigtemplates.bootstrap.cluster.x-k8s.io         2020-01-13T22:26:43Z
machinedeployments.cluster.x-k8s.io                       2020-01-13T22:26:43Z
machines.cluster.x-k8s.io                                 2020-01-13T22:26:44Z
networkpolicies.crd.projectcalico.org                     2020-01-13T22:26:39Z
vsphereclusters.infrastructure.cluster.x-k8s.io           2020-01-13T22:26:44Z
vspheremachines.infrastructure.cluster.x-k8s.io           2020-01-13T22:26:44Z
vspheremachinetemplates.infrastructure.cluster.x-k8s.io   2020-01-13T22:26:45Z
```

Then you can't see them in the side bar.  It looks like 32 characters is the limit for the sidebar.  Can we make it resizable ? 
## Examples of CRD's that work ok 

```
ippools.crd.projectcalico.org  (31 chars)                           2020-01-13T22:26:38Z
machinesets.cluster.x-k8s.io (32 chars)                              2020-01-13T22:26:44Z
```

**Comment by wwitzel3** [comment]
Thanks for reporting this. Yeah, we would expect this to work as well. We will look in to it.

**Comment by wwitzel3** [comment]
Our new navigation (which will be in the next release), shows the entire name of the CRD even when they are very log.

![image](https://user-images.githubusercontent.com/27317/79232663-ce6f7980-7e35-11ea-861b-76015adac17e.png)

closing this as fixed by #812
